jekyll/_cci2/deploy-to-amazon-sagemaker.adoc

```diff
@@ -0,0 +1,364 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.x
+  - Server v3.x
+---
+= Deploy to Amazon SageMaker
+:page-layout: classic-docs
+:page-liquid:
+:page-description: A short page description goes here
+:icons: font
+:experimental:
+
+NOTE: CircleCI releases is currently in **open preview**. During open preview release features are available for free. Charges will be associated with release features once they are generally available. The link:https://circleci.com/developer/orbs/orb/circleci/aws-sagemaker[SageMaker orb] is also in **open preview**.
```

Comment:
Is it worth calling out that the note about potential future charges only applies to CircleCI Releases, not to using the Orb per se?